cli/commands/benchmark-server-timing.mjs

```diff
@@ -83,16 +83,28 @@ export async function handler( opt ) {
 	const { concurrency: connections, number: amount } = opt;
 	const results = [];
 
+	// Log progress only under certain conditions (multiple URLs to benchmark).
+	const logProgress = logURLProgress( opt );
+
 	for await ( const url of getURLs( opt ) ) {
-		const { completeRequests, responseTimes, metrics } = await benchmarkURL(
-			{
-				url,
-				connections,
-				amount,
-			}
-		);
+		if ( logProgress ) {
+			log( `Benchmarking URL ${ url }...` );
```

Comment:
Instead of logging out to STDOUT, can this instead go to STDERR? In that way, if you pipe the output into a file, it will by default not include the logging information. This can be achieved by using `console.warn()` instead of `console.log()`. Or it could use `process.stderr.write()`.

Reply:
What would be the benefit of that? You'd still end up with the "Success." in the file. Or even if we also moved that to STDOUT, the actual errors would end up in the file.

There is probably room to improve this, but if we want to do so, I think we should enhance the existing `log` and `formats` functions available rather than hard-coding in a specific command.

Reply:
The benefit is that the actual output wouldn't get intermixed with the logging. For example, you could do:

```bash
npm run --silent research benchmark-server-timing -- --file=urls.txt --output=csv > results.csv
```

When sending logging information to STDERR, then the CSV file will be pristine and the logging information will only be displayed in the console. Same goes for "Success" below.

And STDERR is not just for errors. It's also for diagnostics. cf. https://stackoverflow.com/a/4919110/93579

Reply:
Makes sense. So I guess we could use either `console.warn()` or `process.stderr.write()`.